proto/babylon/finality/v1/tx.proto

```diff
@@ -103,3 +107,18 @@ message MsgUnjailFinalityProvider {
 
 // MsgUnjailFinalityProviderResponse defines the Msg/UnjailFinalityProvider response type
 message MsgUnjailFinalityProviderResponse {}
+
+// MsgEquivocationEvidence is the message for handling evidence of equivocation
+message MsgEquivocationEvidence {
+    option (cosmos.msg.v1.signer) = "signer";
+
+    string signer = 1;
+    // evidence is the evidence of equivocation
+    Evidence evidence = 2 [(gogoproto.nullable) = false];
+}
+
+// MsgEquivocationEvidenceResponse is the response for MsgEquivocationEvidence
+message MsgEquivocationEvidenceResponse {
+    // success is true if the slashing was successfully applied
+    bool success = 1;
```

Comment:
This is actually not needed because if it fails the server will return error anyway